infra/cmake/packages/TensorFlowConfig.cmake

```diff
@@ -1,5 +1,5 @@
 set(TENSORFLOW_PREFIX "/usr" CACHE PATH "The location of pre-installed TensorFlow library")
-set(TENSORFLOW_VERSION_REQUIRED "1.12.0")
+set(TENSORFLOW_VERSION_REQUIRED "2.3.0")
```

Comment:
I'm not sure about this file... but as of now, our official TF is 1.13.2  :)

Reply:
OK. I'll close this PR. (Unfortunatly, I can't find ensorFlow for C 1.13.2)

Reply:
IMHO, it would be better to introduce TensorFlow-1.12.0 package, and update each component to explicit version.

Here are the projects that depend on TensorFlow.
```
ONE/compiler$ grep -Rn 'find_package' | grep 'TensorFlow'
moco-value-pbtxt-test/CMakeLists.txt:80:nnas_find_package(TensorFlow QUIET)
tfts/CMakeLists.txt:2:nnas_find_package(TensorFlow QUIET)
nnkit-tf/CMakeLists.txt:1:nnas_find_package(TensorFlow QUIET)
gen-core/CMakeLists.txt:7:nnas_find_package(TensorFlow QUIET)
```

Reply:
@hyunsik-yoon Do you have any plan to work on ``gen-core``, ``gen-tf-input``, ``gen-tflite-output``, and ``gen-tf-output``? If not, it would be better to remove these projects ``)

Reply:
``moco-value-pbtxt-test``, ``tfts``, and ``nnkit-tf`` are related with _moco_ (our own TF-to-TFLITE conversion library), which targets TensorFlow 1.13.2.

Reply:
>  I can't find ensorFlow for C 1.13.2

As I know, we can use 1.13.1 for C codes.  1.13.2 was a minor patch.

Reply:
I found `TensorFlow for C 1.13.2` from https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-cpu-linux-x86_64-1.13.2.tar.gz . I'll make PR for `1.13.2` version.

Reply:
@parjong 

> @hyunsik-yoon Do you have any plan to work on gen-core, gen-tf-input, gen-tflite-output, and gen-tf-output? If not, it would be better to remove these projects ``)

#4803 was made.